src/pages/createProfile/createProfileSteps/Complete.tsx

```diff
@@ -0,0 +1,47 @@
+import { FunctionComponent } from "react";
+import { Box } from "@mui/material";
+import NEWMLogo from "assets/images/NEWMLogo";
+import { FilledButton, GradientTypography, Link, Typography } from "elements";
+
+const Complete: FunctionComponent = () => {
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center">
+      <Box mb={ 4 } alignSelf="center">
+        <NEWMLogo />
+      </Box>
+
+      <Typography
+        align="center"
+        fontWeight="extra-bold"
+        variant="xxxl"
+        fontFamily="Raleway"
+      >
+        Aaaaand we&apos;re done.
+      </Typography>
+
+      <Box mt={ 1 } mb={ 6 }>
+        <GradientTypography
+          variant="xxxl"
+          fontFamily="DM Serif Text"
+          fontWeight="regular"
+          fontStyle="italic"
+        >
+          Shall we?
+        </GradientTypography>
+      </Box>
+
+      <FilledButton type="submit">Enter NEWM</FilledButton>
+
+      <Box mt={ 2 }>
+        <Typography fontWeight="medium" color="grey200">
+          By proceeding forward you agree to&nbsp;
+          <Link to="#" color="grey200" fontWeight="medium">
+            projectNEWM’s Terms of Service
```

Comment:
Should the `'` be encoded here too?

Reply:
Probably. It didn't throw an error, but that must be because I copied the text directly from Figma and it's not using an actual apostrophe. I'll update it.